pyproject.toml

```diff
@@ -4,3 +4,112 @@ requires = ["setuptools>=45", "wheel", "setuptools_scm[toml]>=6.0"]
 
 [tool.setuptools_scm]
 write_to = "src/lcm/_version.py"
+
+
+[tool.ruff]
+target-version = "py310"
+fix = true
+select = ["ALL"]
+extend-ignore = [
+    # missing type annotation
+    "ANN001",
+
+    # missing type annotation for `*args`
+    "ANN002",
+
+    # missing type annotation for `**kwargs`
+    "ANN003",
+
+    # missing return type annotation for public function
+    "ANN201",
+
+    # missing return type annotation for private function
+    "ANN202",
+
+    # `zip()` without an explicit `strict` parameter
+    "B905",
+
+    # Missing docstring in public module
+    "D100",
+
+    # missing docstring in public function
+    "D103",
+
+    # missing docstring in public package
+    "D104",
+
+    # exception must not use a string literal
+    "EM101",
+
+    # exception must not use an f-string literal
+    "EM102",
+
+    # boolean default value in function definition
+    "FBT002",
+
+    # Too many arguments to function call
+    "PLR0913",
+
+    # magic value used in comparison
+    "PLR2004",
+
+    # unnecessary variable assignment before return
```

Comment:
Would like to keep it. Trivial to change and doing this

```py
out = whatever(x, y, z)
return out
```

just adds a complication to thinking about code relative to

```py
return whatever(x, y, z)
```

IMHO.